deploy/template.yaml

```diff
@@ -110,22 +110,22 @@ Resources:
             Path: /token
             Method: POST
 
-  IPVAuthorizationFunction:
+  IPVSessionEndFunction:
     Type: AWS::Serverless::Function
     Properties:
-      FunctionName: !Sub "ipv-authorization-${Environment}"
-      Handler: uk.gov.di.ipv.core.authorization.AuthorizationHandler::handleRequest
+      FunctionName: !Sub "ipv-session-end-${Environment}"
+      Handler: uk.gov.di.ipv.core.sessionend.SessionEndHandler::handleRequest
```

Comment:
I've left the session start under the package `uk.gov.di.ipv.core.session`

Happy to change the StartHandler if you want to standardise on one version.